admin/controllers/pupil-register.js

```diff
@@ -18,6 +19,8 @@ const listPupils = async (req, res, next) => {
     next(error)
   }
 
+  const incompletePupils = incompleteChecksPresenter.getPupilWithIncompleteChecks(pupilsFormatted)
```

Comment:
how come we select the pupils and put it in 'pupilsFormatted' then go back for another query just to establish whether any of these pupils have an incomplete status?  could we not do all of that in one query?